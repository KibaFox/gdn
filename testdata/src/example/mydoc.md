# My Document

This is _my_ document with some **Markdown**.
