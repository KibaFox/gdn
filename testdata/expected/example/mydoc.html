<h1>My Document</h1>

<p>This is <em>my</em> document with some <strong>Markdown</strong>.</p>
